packages/contracts/src/Multisig.sol

```diff
@@ -6,6 +6,7 @@ import {SafeCastUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/mat
 
 import {IMembership} from "@aragon/osx-commons-contracts/src/plugin/extensions/membership/IMembership.sol";
 import {Addresslist} from "@aragon/osx-commons-contracts/src/plugin/extensions/governance/Addresslist.sol";
+// solhint-disable-next-line max-line-length
```

Comment:
I think the enable/disable around the whole imports section is personally much cleaner than on a per line basis BUT that's a minor and personal preference. You can ignore if you wish

Reply:
Yes, I thought the same. Changed in 003091042abddd400ef3225e42ba70c21e259c78